plugins/modules/sorcery.py

```diff
@@ -34,34 +32,42 @@
 options:
     name:
         description:
-            - Name of the spell
-            - multiple names can be given, separated by commas
-            - special value '*' in conjunction with states V(latest) or
+            - Name of the spell or grimoire.
+            - Multiple names can be given, separated by commas.
+            - Special value V(*) in conjunction with states V(latest) or
               V(rebuild) will update or rebuild the whole system respectively
-        aliases: ["spell"]
+        aliases: ["spell", "grimoire"]
         type: list
         elements: str
 
+    repository:
+        description:
+            - Repository location.
+            - If specified, O(name) represents grimoire(s) instead of spell(s).
+            - Special value V(*) will pull grimoire from the official location.
+            - Only single item in O(name) in conjunction with V(*) can be used.
+            - O(state) V(absent) must be used with a special value V(*).
+        type: str
+
     state:
         description:
-            - Whether to cast, dispel or rebuild a package
-            - state V(cast) is an equivalent of V(present), not V(latest)
-            - state V(latest) always triggers O(update_cache=true)
-            - state V(rebuild) implies cast of all specified spells, not only
-              those existed before
+            - Whether to cast, dispel or rebuild a package.
+            - State V(cast) is an equivalent of V(present), not V(latest).
+            - State V(rebuild) implies cast of all specified spells, not only
+              those existed before.
         choices: ["present", "latest", "absent", "cast", "dispelled", "rebuild"]
         default: "present"
         type: str
 
     depends:
         description:
             - Comma-separated list of _optional_ dependencies to build a spell
-              (or make sure it is built) with; use +/- in front of dependency
-              to turn it on/off ('+' is optional though).
-            - this option is ignored if O(name) parameter is equal to V(*) or
+              (or make sure it is built) with; use V(+)/V(-) in front of dependency
+              to turn it on/off (V(+) is optional though).
+            - This option is ignored if O(name) parameter is equal to V(*) or
               contains more than one spell.
-            - providers must be supplied in the form recognized by Sorcery, for example
-              'openssl(SSL)'.
+            - Providers must be supplied in the form recognized by Sorcery,
+              for example 'V(openssl(SSL))'.
```

Comment:
Change line 13 to `DOCUMENTATION = r'''` (with the `r`) and change this to
```suggestion
              for example 'V(openssl(SSL\))'.
```